diag_manager/fms_diag_yaml.F90

```diff
@@ -1457,6 +1458,22 @@ function get_diag_fields_entries(indices) &
 
 end function get_diag_fields_entries
 
+!> @brief Gets the output buffer ids corresponding to the indices in the sorted variable_list
+!! @return Array of indices of the output buffers
+function get_output_buffer_ids(indices) result(buffer_ids)
```

Comment:
The name of this function is misleading. This returns the indices of the field in the `diag_yaml%diag_fields` object. It is not related to the buffer ids. 

https://github.com/NOAA-GFDL/FMS/blob/b9b210342da7ae3248519b17ce3ce77419b8bbe5/diag_manager/fms_diag_yaml.F90#L228

Reply:
@uramirez8707 Can you comment on this?